tests/storage/test_event_push_actions.py

```diff
@@ -312,7 +312,7 @@ def _create_event(
         def _rotate() -> None:
             self.get_success(self.store._rotate_notifs())
 
-        def _mark_read(event_id: str, thread_id: Optional[str] = None) -> None:
+        def _mark_read(event_id: str, thread_id: str = "main") -> None:
```

Comment:
Using `"main_thread"` as the special string seems like it would be more clear when it's used below as an unnamed arg, `_mark_read(last_event_id, "main_thread")`

Reply:
Related to https://github.com/matrix-org/synapse/pull/13877#discussion_r982009075

Reply:
Per https://github.com/matrix-org/synapse/pull/13877#discussion_r982009075, it seems like we just need to use the constant throughout which has a nice name as well

```suggestion
        def _mark_read(event_id: str, thread_id: str = MAIN_TIMELINE) -> None:
```

Reply:
I think I just failed to use the constant on this PR, yeah. Thanks!